src/image-viewer.js

```diff
@@ -316,15 +316,18 @@ export class ImageViewer {
 
   /** @private */
   setupGestures_() {
-    this.gestures_ = Gestures.get(this.image_);
+    this.gestures_ = Gestures.get(this.viewer_,
+        /* opt_shouldNotPreventDefault */true);
     this.gestures_.onPointerDown(() => {
       if (this.motion_) {
+        event.preventDefault();
         this.motion_.halt();
       }
     });
 
     // Zoomable.
     this.gestures_.onGesture(DoubletapRecognizer, e => {
+      event.preventDefault();
```

Comment:
`e.`?

Reply:
This is to prevent triggering browser default zoom. iOS safari also has double tap zoom on by default, so if we don't call `event.preventDefault()` here, it will trigger both our custom zoom as well as safari's default zoom.